test/utils/describeConformanceUnstyled.tsx

```diff
@@ -75,36 +75,36 @@ function testPropForwarding(
 
   it('forwards custom props to the root element if a component is provided', () => {
     const CustomRoot = React.forwardRef(
-      ({ fooBar, tabIndex }: WithCustomProp, ref: React.ForwardedRef<any>) => {
+      ({ fooBar, role }: WithCustomProp, ref: React.ForwardedRef<any>) => {
```

Comment:
The `tabIndex` in the `Slider` was being passed to the hidden input element, so this test was failing. I changed the attribute for now, but maybe we should also only test `fooBar` here, as another component may have the same problem. cc @michaldudak up to you.

Reply:
I'm afraid `role` may suffer from the same issue. We could use some more exotic HTML property such as `lang` instead. I don't think any of our components would set it.